src/Daemon/BaseDaemon.cpp

```diff
@@ -336,30 +324,22 @@ class SignalListener : public Poco::Runnable
         else if (sig >= 0)
             signal_description = strsignal(sig); // NOLINT(concurrency-mt-unsafe) // it is not thread-safe but ok in this context
```

Comment:
Let's also replace it with `sigdescr_np` just in case

Reply:
Looks like we don't have `sigdescr_np` in our contirbs.
I've rearranged the code to print signal number first.